README.md

```diff
@@ -46,7 +46,9 @@ If your app depends on the old unmigrated atmosphere packages than you also need
 
 ## The simplest possible test
 
-Basically, you run the tests with `gagarin` command within you project root.
+First run your meteor app with command `meteor`. 
```

Comment:
Strange thing happen...
Whatever I do ( version 0.4.5 or 0.4.6 ) gagrin is hanging on build

➜  gagarin-example-the-simplest-one git:(master) ✗ gagarin --verbose

  --- building app => /Users/me/projects/gagarin/gagarin-example-the-simplest-one ---